src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -840,17 +840,7 @@ var FormValidator = A.Component.create({
                     }
                 }
                 else if (!field.val()) {
-                    namedFieldNodes.each(
-                        function(node, index, nodeList) {
-                            node.removeClass('errorClass');
-                            node.removeAttribute('aria-invalid');
-                        }
-                    );
-
-                    if (fieldContainer) {
```

Comment:
I believe this takes care of it: https://github.com/mairatma/alloy-ui/pull/340/commits/d8edace88af9f91d0cfec5ec172b3724534b1b1d